external/SPHEREPACK/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+enable_language(Fortran)
```

Comment:
Please just add `Fortran` to the `project` command in the highest level CMake file, instead of this here. It would be worth adding a comment above `project` stating the Fortran is required because of SPHEREPACK